src/api/apps/users/model.coffee

```diff
@@ -52,7 +52,7 @@ jwtDecode = require 'jwt-decode'
 save = (user, accessToken, callback) ->
   async.parallel [
     (cb) ->
-      db.collection('users').findOne {_id: ObjectId(user.id)}, cb
+      db.collection('users').findOne {user_ids: ObjectId(user.id)}, cb
```

Comment:
Could you elaborate a bit about this change? I am unclear about the change from `_id` to `user_ids` for the user lookup by id.

Reply:
This was actually the original logic which was mistakenly [changed](https://github.com/artsy/positron/commit/fada33157ff1d80afe36cd307595c9112834bb84#diff-47ba709974806d6b0cb616d465bfb8a8f3e7d5dca78f2afdcfd0c38a92407e77L55) in #3131. This just corrects that change.